test/integration/test_open_analyse_save_load_project.c

```diff
@@ -44,15 +44,12 @@ bool test_open_analyse_save() {
 	mu_assert_eq(err, RZ_PROJECT_ERR_SUCCESS, "project save err");
 
 	// 5. Close the file
-	rz_core_file_free(file);
+	rz_core_file_close(file);
 	rz_core_free(core);
 
-	// 6. Open the file again
+	// 6. Create a new core
 	core = rz_core_new();
 	mu_assert_notnull(core, "new RzCore instance");
```

Comment:
Is this removal for a purpose? Why it was left as is on the first opening but you removed it after?

Reply:
The second time you are loading a project with the `load_bin_io = true` arg. This will reload the file automatically so loading the file manually the second time is not necessary and it would just be overwritten.